code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -27,12 +27,20 @@
 	use_power = IDLE_POWER_USE
 	idle_power_usage = 5
 	layer = BELOW_OBJ_LAYER
-	var/obj/item/reagent_containers/food/snacks/deepfryholder/frying	//What's being fried RIGHT NOW?
+	/// What's being fried RIGHT NOW?
+	var/obj/item/reagent_containers/food/snacks/deepfryholder/frying
+	/// Who's bring fried RIGHT NOW?
+	var/obj/item/clothing/head/mob_holder/frying_mob
```

Comment:
Still conflicted about this

Reply:
Have you seen the current state of this?
https://user-images.githubusercontent.com/39249137/152250368-80910e83-f94c-49b7-bc10-7ec66dc285ff.mp4

Reply:
I more meant that I was conflicted about them being truly fryable *at all,* not that it's a regression or anything.

Reply:
I'm generally of the disposition that it's better to change the outcome of an option than removing the option. The problem with deepfrying held mobs isn't the act itself, but the fact that it has silly behaviour and is of no real consequence to both parties.

You mentioned an issue with the current state of kepori:
> ...most significantly them being taken seriously as a race.

This PR was made to resolve that make actually taking the idea seriously and making the actions have consequence to consider.

Reply:
i think as long as the frying is mechanically the same as the normal neck-grab fry, it should be fine, since it just makes an unintended possibility work in line with similar actions